meshtastic/mqtt.proto

```diff
@@ -31,3 +31,17 @@ message ServiceEnvelope {
    */
   string gateway_id = 3;
 }
+
+/*
+ * This message will be proxied over the PhoneAPI for client to deliver to the MQTT server
+ */
+message MqttClientProxyMessage {
+  /*
+   * The MQTT topic this message will be sent /received on
+   */
+  string topic = 1;
+  /*
```

Comment:
Do we really need this? The PhoneAPI will get the config packets when starting up (want_config) and the custom MQTT topic root is in the config, if set. If not we can just opt to the default msh/2/c/<relayid> where relayid comes from the envelope.

Reply:
My initial plan was to construct the full topic for the message we're sending up, so that the client can be completely dumb and just lob a serialized ServiceEnvelope at that topic on MQTT. We can omit it though and place the responsibility on the client though.